pos_partner_birthdate/static/src/js/ClientDetailsEdit.esm.js

```diff
@@ -1,15 +1,16 @@
 /** @odoo-module **/
+import {useState} from "@odoo/owl";
```

Comment:
You've got `owl` as a global object
```suggestion
const {useState} = owl;
```